cpp/daal/src/threading/threading.cpp

```diff
@@ -111,6 +111,33 @@ DAAL_EXPORT void _daal_threader_for(int n, int threads_request, const void * a,
 #endif
 }
 
+DAAL_EXPORT void _daal_static_threader_for(size_t n, const void * a, daal::functype_static func)
+{
+#if defined(__DO_TBB_LAYER__)
+    const size_t nthreads           = _daal_threader_get_max_threads();
```

Comment:
`_daal_threader_get_max_threads` Will return the maximum number of threads, even when we have limited for example one thread?

Reply:
It's just bad name of the function.

It returns # to use threads. So, the function will return the number what you set.